src/_pytest/assertion/util.py

```diff
@@ -102,38 +103,45 @@ def _format_lines(lines):
     basestring = str
 
 
-def assertrepr_compare(config, op, left, right):
-    """Return specialised explanations for some operators/operands"""
-    width = 80 - 15 - len(op) - 2  # 15 chars indentation, 1 space around op
-    left_repr = saferepr(left, maxsize=int(width // 2))
-    right_repr = saferepr(right, maxsize=width - len(left_repr))
+def issequence(x):
+    return isinstance(x, Sequence) and not isinstance(x, basestring)
+
+
+def istext(x):
+    return isinstance(x, basestring)
+
+
+def isdict(x):
+    return isinstance(x, dict)
 
-    summary = u"%s %s %s" % (ecu(left_repr), op, ecu(right_repr))
 
-    def issequence(x):
-        return isinstance(x, Sequence) and not isinstance(x, basestring)
+def isset(x):
+    return isinstance(x, (set, frozenset))
 
-    def istext(x):
-        return isinstance(x, basestring)
 
-    def isdict(x):
-        return isinstance(x, dict)
+def isdatacls(obj):
+    return getattr(obj, "__dataclass_fields__", None) is not None
 
-    def isset(x):
-        return isinstance(x, (set, frozenset))
 
-    def isdatacls(obj):
-        return getattr(obj, "__dataclass_fields__", None) is not None
+def isattrs(obj):
+    return getattr(obj, "__attrs_attrs__", None) is not None
 
-    def isattrs(obj):
-        return getattr(obj, "__attrs_attrs__", None) is not None
 
-    def isiterable(obj):
-        try:
-            iter(obj)
-            return not istext(obj)
-        except TypeError:
-            return False
+def isiterable(obj):
+    try:
+        iter(obj)
+        return not istext(obj)
+    except TypeError:
+        return False
```

Comment:
This is moved out of the function only, which I'd like to have done anyway - but allows for easy monkeypatching.